util/posix_close.h

```diff
@@ -0,0 +1,17 @@
+#ifndef _POSIX_CLOSE_H_
+#define _POSIX_CLOSE_H_
+
+/*
+ * Use a #define in order to avoid namespace collisions with anyone else's
+ * posix_close() (e.g., musl).
+ */
+#define posix_close libcperciva_posix_close
+
+/**
+ * posix_close(fd, flags):
+ * Close a descriptor, but looping until completion if interrupted by a signal.
+ * ${flags} must be 0.
```

Comment:
This aims to match the upcoming `posix_close()` function:
https://www.austingroupbugs.net/view.php?id=529
comment 0001200.

Reply:
Hmm.  I was thinking that this should be grouped with the other "noeintr" functions; it's not exactly a drop-in replacement for `posix_close` since it's *way* more complicated.  Let me think about this...